lib/runner/test-runner/test-runner.js

```diff
@@ -63,29 +69,27 @@ module.exports = testRunner = {
             if (testRunner._rerunCount === 0) {
               process.exitCode = 1;
             }
-            testRunner._testsToRerun.push(test.testPath);
+            testRunner._testRerunMemo[test.testPath]--;
+            if (testRunner._testRerunMemo[test.testPath] >= 0) {
+              testRunner._testFiles.push(test.testPath);
+              testRunner.on('testRunner.configured', testRunner._scheduleTests);
```

Comment:
I'm trying figure out why this line of code is here. Can you explain it to me? I forgot a lot of this codebase.

Reply:
If there are any reties left in the map, then after the tests are queued into `testFiles` they need to be scheduled to be executed

Reply:
if the memo has retries left, then they are queued into the `testFiles` list. From there the contents of the queue have to be scheduled to be executed. firing `testRunner.configured` calls the function that does that

Reply:
It was a mistake in a couple of ways. 

1. It was supposed to be testRunner.emit(testRunner.configured) to fire the tests on queue
2. testRunner.emit wasnt needed to empty the queue